homeassistant/components/script/config.py

```diff
@@ -1,20 +1,66 @@
 """Config validation helper for the script integration."""
 import asyncio
+from contextlib import suppress
 
 import voluptuous as vol
 
-from homeassistant.config import async_log_exception
-from homeassistant.const import CONF_SEQUENCE
+from homeassistant.components.blueprint import (
+    BlueprintInputs,
+    is_blueprint_instance_config,
+)
+from homeassistant.config import async_log_exception, config_without_domain
+from homeassistant.const import (
+    CONF_ALIAS,
+    CONF_DEFAULT,
+    CONF_DESCRIPTION,
+    CONF_ICON,
+    CONF_NAME,
+    CONF_SELECTOR,
+    CONF_SEQUENCE,
+    CONF_VARIABLES,
+)
 from homeassistant.exceptions import HomeAssistantError
-from homeassistant.helpers import config_validation as cv
-from homeassistant.helpers.script import async_validate_action_config
-
-from . import DOMAIN, SCRIPT_ENTRY_SCHEMA
+from homeassistant.helpers import config_per_platform, config_validation as cv
+from homeassistant.helpers.script import (
+    SCRIPT_MODE_SINGLE,
+    async_validate_action_config,
+    make_script_schema,
+)
+from homeassistant.helpers.selector import validate_selector
+
+from .const import CONF_ADVANCED, CONF_EXAMPLE, CONF_FIELDS, CONF_REQUIRED, DOMAIN
+from .helpers import async_get_blueprints
+
+PLATFORM_SCHEMA = make_script_schema(
```

Comment:
This is not a valid `PLATFORM_SCHEMA`. The name implies it's the configuration for a platform but in this case it's the validation for a Script entity. I would call it `SCRIPT_ENTITY_SCHEMA` to avoid confusion with `PLATFORM_SCHEMA`. `PLATFORM_SCHEMA` also has actual meaning since it can influence config pre-processing by Home Assistant.

Reply:
I guess the old name was `SCRIPT_ENTRY_SCHEMA`, which also works.

Reply:
Adjusted.